src/snovault/tests/test_auditor.py

```diff
@@ -133,3 +133,40 @@ def test_link_target_audit_pass(testapp):
         errors_list.extend(errors_dict[error_type])
     errors = [e for e in errors_list if e['name'] == 'testing_link_target_status']
     assert errors == []
+
+
+def test_audit_inherit_error(testapp):
+    target = {'uuid': '775795d3-4410-4114-836b-8eeecf1d0c2f', 'status': 'CHECK'}
+    testapp.post_json('/testing_link_target', target, status=201)
+    source = {'uuid': '16157204-8c8f-4672-a1a4-14f4b8021fcd', 'target': target['uuid'], 'condition1': True}
+    # condition1 - auto fails the audit on source
+    testapp.post_json('/testing_link_source', source, status=201)
+    
+    res = testapp.get('/%s/@@index-data' % target['uuid']).maybe_follow()
+    # index TARGET
+    errors_dict = res.json['audit']
+    errors_list = []
+    for error_type in errors_dict:
+        errors_list.extend(errors_dict[error_type])
+    errors = [e for e in errors_list if e['name'] == 'checker1']
+    error, = errors
+    assert error['detail'] == 'Missing checker1'
+    assert error['category'] == 'testchecker'
+    assert error['level'] == 0
+    assert error['path'] == '/testing-link-sources/16157204-8c8f-4672-a1a4-14f4b8021fcd/'
+
+
+def test_audit_inherit_no_error(testapp):
+    target = {'uuid': '775795d3-4410-4114-836b-8eeecf1d0c2f', 'status': 'CHECK'}
+    testapp.post_json('/testing_link_target', target, status=201)
+    source = {'uuid': '16157204-8c8f-4672-a1a4-14f4b8021fcd', 'target': target['uuid'], 'condition1': False}
+    testapp.post_json('/testing_link_source', source, status=201)
+    
+    res = testapp.get('/%s/@@index-data' % target['uuid']).maybe_follow()
+    errors_dict = res.json['audit']
+    errors_list = []
+    for error_type in errors_dict:
+        errors_list.extend(errors_dict[error_type])
```

Comment:
I don't think we need errors or errors_list.  Can't we just assert errors_dict[error_type]['name'] != 'checker1'
```
errors_list = []
for error_type in errors_dict:
    assert not errors_dict[error_type]['name'] == 'checker1'
```

Reply:
I'll try it.